### ISSUE TITLE
Reg\Operation - Opt-ins with existing BORO IDs are not shown Opt-In application

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#2398]
#### Dependencies

- Blocked by
- Blocking

#### Description:

Some opt-in operations already received a BORO ID, so when they come back in registration 2 to register their operation, they should NOT be shown the opt-in application form since they already filled it out before. [Thread](https://teams.microsoft.com/l/message/19:143b7487e8a34e28b0660f9e39d8d502@thread.tacv2/1729197497096?tenantId=6fdb5200-3d0d-4a8a-b036-d3685e359adc&groupId=29821293-3244-455d-beae-e01d27762cfe&parentMessageId=1729197497096&teamName=External%3A%20CAS%20-%20Clean%20Growth%20Digital%20Service%20Team&channelName=Carbon%20Pricing%20Transition%20SMEs&createdTime=1729197497096&ngc=true) from SME channel for context. 

#### Acceptance Criteria:

Given that my operation is an Opt-in with a BORO ID
When I am registering this operation
Then I am not shown the opt-in application

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

**Comment by patrickisaac** [comment]
Refinement Notes:
- @andrea-williams to add Dev checklist

**Comment by andrea-williams** [comment]
@zoeyli-46  would I be correct in assuming that the "Opted in Operation" section of the Operation form in view mode (https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=18025-1179&m=dev&t=HNQ6XaR8QUjb2CZ2-1 ) should not appear in this case? What about for Edit mode of the form?

**Comment by zoeyli-46** [comment]
Based on this [thread](https://teams.microsoft.com/l/message/19:143b7487e8a34e28b0660f9e39d8d502@thread.tacv2/1729634671401?tenantId=6fdb5200-3d0d-4a8a-b036-d3685e359adc&groupId=29821293-3244-455d-beae-e01d27762cfe&parentMessageId=1729197497096&teamName=External%3A%20CAS%20-%20Clean%20Growth%20Digital%20Service%20Team&channelName=Carbon%20Pricing%20Transition%20SMEs&createdTime=1729634671401&ngc=true), because Opted-in operations from registration 1 submitted the application manually outside of BCIERS, their application data will not be showing up in BCIERS. Therefore there will not be an opted in operation section of the operation form for these operations. 

For future consideration: would the business area want the operations to have the functionality to add their opt-in application data in the Operation Information form in edit mode? So, they wont see the form in Register Operation BUT when they go to edit their [Operation Information form](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=18025-1336&t=qce0ljC66QKbaLxt-4) then they could see a blank opt-in form that they could fill in? is this confusing for the user because they don't understand why it would be empty? is this difficult to implement anyways?

**Comment by andrea-williams** [comment]
controversial opinion: would it be acceptable to have previously opted-in operators fill out the opt-in form again (this time within BCIERS)? It's just a few yes/no questions, it shouldn't be a lot of time or effort for them to fill it out again.

**Comment by patrickisaac** [comment]
Refinement Notes:
- Potential discussion later during Reg. Collab (Nov. 5)

**Comment by patriciarussellCAS** [comment]
Just looping back on this and re-adding the backlog refinement tag. It was discussed at Reg Collab on Nov 5 and BA and DST came to the following decision: 

- for now we will plan to require already designated opt-ins to complete the opt-in form again (unless we have time to go back and fix it later). 
- It was agreed that we would add additional communications to explain how this works before we launch. 

cc: @zoeyli-46 @andrea-williams
